packages/framework/src/Resources/translations/messages.cs.po

```diff
@@ -100,6 +100,9 @@ msgstr "Přidat kombinaci jazyků"
 msgid "Add limit"
 msgstr "Přidat limit"
 
+msgid "Add new customer user"
+msgstr "Přidat nový zákaznícky účet"
```

Comment:
```suggestion
msgstr "Přidat nový zákaznický účet"
```

Reply:
All translations fixed in 
- https://github.com/shopsys/shopsys/pull/3261/commits/168a34f67d07eef72be2f4ca90242a32653e6d5f
- https://github.com/shopsys/shopsys/pull/3261/commits/b14c66839bc0b30772659fd66abc0c32a24e20d9